src/app/content/highlights/components/HighlightsPopUp.tsx

```diff
@@ -70,9 +83,9 @@ class HighlightsPopUp extends Component<Props> {
           </Styled.GeneralTextWrapper>
           <Styled.MyHighlightsImage src={myHighlightsEmptyImage} />
         </Styled.MyHighlightsWrapper>
-      </Styled.PopupBody>
+      </React.Fragment>
```

Comment:
you could just return this wrapped in the PopupBody, no need for the fragment and then wrap it in myHighlights